From 6797a2a6b8ce1d189104502ab57758eb454d1b32 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 23 Jul 2013 22:01:39 -0400
Subject: [PATCH 102/108] initctl: use irreversible jobs when switching
 runlevels

Spotted by uau in #systemd.

(cherry picked from commit 4c10771ce273872f4833da4e9dd7e1c72cf20de5)
---
 src/initctl/initctl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/initctl/initctl.c b/src/initctl/initctl.c
index 735f1e1..5fbce4a 100644
--- a/src/initctl/initctl.c
+++ b/src/initctl/initctl.c
@@ -122,7 +122,7 @@ static void change_runlevel(Server *s, int runlevel) {
         if (isolate)
                 mode = "isolate";
         else
-                mode = "replace";
+                mode = "replace-irreversibly";
 
         log_debug("Running request %s/start/%s", target, mode);
 
-- 
1.8.3.4

